game_test.py

```diff
@@ -0,0 +1,99 @@
+import chess
+import unittest
+from game import *
+
+
+class FlaskTestCase(unittest.TestCase):
```

Comment:
Why is the name of the class "FlaskTestCase" when you test the game mechanics with the chess library and not Flask. But i guess that's just a typo from copying.